snap/wrappers/juju

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export LXD_DIR=/var/snap/lxd/common/lxd
```

Comment:
Does this mean we only work with snapped lxd, what is the story between debian lxd and snap lxd? Do they see the same containers?